### DISCUSSION TITLE
Passaggio da Home Assistant OS a Core: come "migrare" la skill di Alexa?

### DISCUSSION THREAD

**Discussion by axistant** [indomus/forum#2421] [HUB: Home Assistant]
Ciao a tutti. Ho recentemente deciso di passare a Core, dopo un annetto di esperienza in Hassio. Ho preso un mini PC, installato Debiano, HA, Mosquitoo, Zigbee2MQTT (grazie alle stupende guide di Indomus). Prima di effettuare il passaggio dei dispositivi, mi sono chiesto se per caso non ci fosse una "scorciatoia" per integrare Alexa via Haaska e AWS senza effettuare da capo tutta la procedura.
"Vagabondando" sul sito AWS Lambda, ho visto che nel file config.json ci sono l'indirizzo del vecchio sito di Hassio e il token di lunga durata: non è che per caso modificando questi due valori si riesce a far puntare il tutto verso la nuova installazione?
La ragione mi dice di no, ma il cuore mi ha spinto a fare una domanda pazza...

**Comment by snakuzzo** [comment]
E invece hai centrato la soluzione :)
Ti basta nel caso adeguare FQDN e access token (se non ti porti dietro quanto presente sotto `.storage` ti basterà crearne uno nuovo), salvare, cliccare su `DEPLOY` ed il gioco sarà fatto.

**Reply by axistant** [reply]
Grazie mille! Mi sembrava troppo bello per essere vero, e invece... Questo mi farà risparmiare un bel po' di tempo, grazie ancora!
